internal/explorer/config_explorer.go

```diff
@@ -81,11 +81,18 @@ func (e configExplorer) FindResources() (map[string]Resource, error) {
 			continue
 		}
 
+		parameters, err := extractParameters(e.spec.Paths, resourceConfig.Read.Path)
+		if err != nil {
+			errResult = errors.Join(errResult, fmt.Errorf("failed to extract '%s.delete': %w", name, err))
```

Comment:
Looks like this error message got copied over. We can potentially return a message like:

```suggestion
			errResult = errors.Join(errResult, fmt.Errorf("failed to extract '%s' common parameters: %w", name, err))
```